lib/Fencer/Time.hs

```diff
@@ -27,7 +27,7 @@ newtype Timestamp = Timestamp Int64
 
 -- | Get current time as a 'Timestamp'.
 getTimestamp :: IO Timestamp
-getTimestamp = Timestamp . systemSeconds <$> getSystemTime
```

Comment:
This should also provide a short explanation re/ why we are using `clock`.

Reply:
Done.